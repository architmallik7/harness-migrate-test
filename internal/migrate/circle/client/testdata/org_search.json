[
    {
        "vcs_type": "circleci",
        "slug": "circleci/2KYrz6YMknqk6SPrjhdfnR",
        "name": "Ash Williams-we66x",
        "id": "0ab0297c-3cba-4be9-9bcc-6fd7f71c5242",
        "avatar_url": "https://avatars.githubusercontent.com/u/1231870?v=4"
    },
    {
        "vcs_type": "circleci",
        "slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS",
        "name": "Ash Williams",
        "id": "81700eff-acc7-459b-9c17-2f9e3d937db7",
        "avatar_url": "https://avatars.githubusercontent.com/u/1231870?v=4"
    }
]