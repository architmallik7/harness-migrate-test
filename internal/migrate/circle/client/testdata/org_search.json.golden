{
  "id": "81700eff-acc7-459b-9c17-2f9e3d937db7",
  "type": "circleci",
  "name": "Ash Williams",
  "slug": "circleci/Gz3a4NZuvL2Eng3EFmqdLS"
}